naga/src/back/spv/helpers.rs

```diff
@@ -85,7 +85,7 @@ impl crate::AddressSpace {
 
 /// Return true if the global requires a type decorated with `Block`.
 ///
-/// Vulkan spec v1.3 §15.6.2, "Descriptor Set Interface", says:
+/// In the Vulkan spec, the section [Descriptor Set Interface][dsi] says:
```

Comment:
question(non-blocking): I think it's good to avoid presenting details that need explicit maintenance like versions and section numbers if they can be encoded into a link, but I just wanted to check: was that intentional? Was this section number maybe incorrect?

Reply:
I believe what happened in this case was that the section number actually *changed*. I assume the Vulkan spec editors try to keep fragment ids stable. So yes, maybe just dropping the section number would be best.

However, I think the version number is relevant. The Vulkan spec web site doesn't seem to support permalinks, so if a future version changes the language I'm quoting substantially, it seems helpful to let people know what I was actually quoting. But in that case, a full three-component spec version would be better.

So: I'll include a detailed version number, and delete the section number, and try to follow that practice elsewhere in the future.

Reply:
I think I misremembered what this commit actually does.

1) Yes, the section number was incorrect. I believe this is because the spec changed, so to avoid having to revisit this I switched to the fragment ID, which I assume will be more stable.

2) I deleted the version number because it seemed like the link covered that. But as I said in the prior comment, in the absence of a permalink facility I can imagine it being valuable to actually say what exact version of the doc I was looking at. The link doesn't cover that.